src/main/java/com/festa/service/AccountsService.java

```diff
@@ -46,14 +42,6 @@ public void signUp(SignUpDTO signUpDTO) {
         }
     }
 
-    /**
-     * 로그인
-     *
-     * 아이디가 존재하는지 확인 후 존재하지 않는다면 IsNotExistedException 발생하며 중단
-     *
-     * @param loginDTO
-     * @return
-     */
     public void login(LoginDTO loginDTO, HttpSession httpSession){
```

Comment:
띄어쓰기 하나 넣으면 좋을 것 같아요~

```suggestion
    public void login(LoginDTO loginDTO, HttpSession httpSession) {
```

Reply:
네~